infoview/extension.ts

```diff
@@ -1,50 +1,86 @@
+/* This file contains everything that is specific to the vscode extension
+implementation of the infoview. So the idea is that lean-web-editor
+shares the rest of this infoview directory with this project. */
+
+import * as trythis from '../src/trythis';
+export {trythis};
```

Comment:
You could also move the regexps in the `trythis` module to the infoview (or shared) directory and then import them the other way around from the `trythis.ts` file.

I'd like to avoid multiple copies of the regexes if possible.

Reply:
I don't quite get what you mean by this, but I don't understand what 'try this' is doing either.

Reply:
If you call `squeeze_simp`, then it will print `Try this: simp only [foo]`.  You can then click on the "Try this" in the infoview and it will insert the suggestion in the editor.  We also have a code action for this, which is contained in `tacticsuggestions.ts`.  The `trythis.ts` module contains the regexps used to parse the suggestion.

Right now, the infoview imports the regexps from `src/trythis.ts`.  My proposal is to move `trythis.ts` to `infoview/trythis.ts` and have `tacticsuggestions.ts` import `infoview/trythis.ts` instead.  Then we would only require one copy of this file.

Reply:
There's only one copy of the file at the moment though.

Reply:
I assumed that you would copy the file to the web editor.

Reply:
ah I see what you mean now